<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton_start">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>780</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_stop">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>780</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_water">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_target">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Target = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mortar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mortar temperature = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_power">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Water = </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Static">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>720</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Static target:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Matching">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>750</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matching. Gap between Setpoint and Input:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Static">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>720</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>80.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="value">
     <double>18.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Matching">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>750</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-0.500000000000000</double>
    </property>
    <property name="maximum">
     <double>0.500000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_calibration">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>690</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calibration</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_ramp">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>660</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ramp test</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_power">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1202</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
